frontend/__tests__/src/fixtures.ts

```diff
@@ -388,17 +388,17 @@ export const EXPECTED_REPORT_VALUES = {
       valuesList: [
         {
           name: 'Deployment frequency(deployments/day)',
-          value: '30.00%',
+          value: '0.3',
```

Comment:
it is a string? should be a number

Reply:
The value in three column table UI structure is string.  
Because the three column table needs to extract a common UI data structure, in order to maintain uniformity, it is set to string.